### ISSUE TITLE
Resizing the crop frame in Aspect Ratio mode like Google Photos

### ISSUE THREAD

**Issue by bsmn99** [Yalantis/uCrop#261]
It would be better if the crop frame after selecting the aspect ratio is re sizable like in present google photos app crop feature, since user can resize thorough corners to exact the location of picture to crop it, even though pinch/zoom works if resize cropping may add like enhancement. I am in need of that functionality.

**Comment by TeeRawk** [comment]
Hello @bsmn99  , I just checked that this feature is currently available in UCrop. Can you check this ?

**Comment by bsmn99** [comment]
@TeeRawk No, u may answering based on free cropping in ucrop, pls check with google photos after selecting the aspect ratio in that..
